### PR TITLE
Importexport

### PR DESCRIPTION
**Pull request by allanaaa**
Making a PR because I forgot about this branch ages ago.

The idea is to consolidate all getting-data-in and getting-data-out information on one page for convenience. Helpful to review what might need further development too, or we could add suggestions for external tools (for, say, copying a website nicely, etc.).

### CODE REVIEW

**Comment by sharonmleon** [comment]
I commented in line on this, but I'm not sure that it was apparent, @allanaaa

**Comment by allanaaa** [comment]
> I commented in line on this, but I'm not sure that it was apparent, @allanaaa

You might need to do something further to "submit" the review? I'm not seeing anything yet.

**Review by sharonmleon — COMMENTED** [review]
That was it. I didn't finish the review. Sorry.

File: docs/importexport.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+This script cannot export site or page information, nor will it export resource templates, vocabularies, or other data stored in modules (such as Inverse Properties settings on resource templates). 
+
+#### Requirements
+
+omeka-s-csv.py works with either Python 2 or 3, and requires no additional or external dependencies.
+
+Linux users will likely have Python already installed, and packages should be available from your distribution. There are several options for installing Python on Windows and Mac, including downloading an installer [from Python's website](https://www.python.org/downloads/).
+
+#### Export
```
[the remark this answers is not in the document]
Comment (allanaaa) [reply docs/importexport.md:63]: What do you mean by this? Which other output formats?

[the remark this answers is not in the document]
Comment (jimsafley) [reply docs/importexport.md:63]: Clients can specify the output format when making requests against the API, by adding the query: `format=<format>`. Here are the available formats:

- `jsonld` (default)
- `n3`
- `ntriples`
- `rdfxml`
- `turtle`

Modules may set more formats using the `api.output.formats` and `api.output.serialize` events.

[the remark this answers is not in the document]
Comment (allanaaa) [reply docs/importexport.md:63]: Ah. This is in the developer docs, so I'll link there.

**Pull request merged**